imports/plugins/core/dashboard/client/containers/plugin-versions.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import PluginVersion from "../components/plugin-versions";
+
+class PluginVersionsContainer extends Component {
```

Comment:
In new code we name the file the same as the component, in the same case: `PluginVersionsContainer.js`